reflex/state.py

```diff
@@ -1093,6 +1093,9 @@ def __setattr__(self, name: str, value: Any):
         # Set the var on the parent state.
         inherited_vars = {**self.inherited_vars, **self.inherited_backend_vars}
         if name in inherited_vars:
+            if self.get_name() == "frontend_event_exception_state":
+                return
```

Comment:
Because when running Reflex with Redis, I get the error below. Without Redis, everything works fine. Do you know why? You can test it in the integration/test_exception_handlers.py

```bash

ERROR    asyncio:base_events.py:1821 Task exception was never retrieved
future: <Task finished name='Task-32' coro=<AsyncServer._handle_event_internal() done, defined at /Users/max/Library/Caches/pypoetry/virtualenvs/reflex-D4pZYT2f-py3.12/lib/python3.12/site-packages/socketio/async_server.py:605> exception=AttributeError("'NoneType' object has no attribute 'router'")>
Traceback (most recent call last):
  File "/Users/max/Library/Caches/pypoetry/virtualenvs/reflex-D4pZYT2f-py3.12/lib/python3.12/site-packages/socketio/async_server.py", line 607, in _handle_event_internal
    r = await server._trigger_event(data[0], namespace, sid, *data[1:])
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/max/Library/Caches/pypoetry/virtualenvs/reflex-D4pZYT2f-py3.12/lib/python3.12/site-packages/socketio/async_server.py", line 643, in _trigger_event
    return await handler.trigger_event(event, *args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/max/Library/Caches/pypoetry/virtualenvs/reflex-D4pZYT2f-py3.12/lib/python3.12/site-packages/socketio/async_namespace.py", line 37, in trigger_event
    ret = await handler(*args)
          ^^^^^^^^^^^^^^^^^^^^
  File "/Users/max/projects/senbax/reflex-maxim/reflex/app.py", line 1514, in on_event
    async for update in process(self.app, event, sid, headers, client_ip):
  File "/Users/max/projects/senbax/reflex-maxim/reflex/app.py", line 1273, in process
    state.router = RouterData(router_data)
    ^^^^^^^^^^^^
  File "/Users/max/projects/senbax/reflex-maxim/reflex/state.py", line 1099, in __setattr__
    setattr(self.parent_state, name, value)
AttributeError: 'NoneType' object has no attribute 'router'

```

Reply:
that error arises because the event is triggered without the parent state prefix

* `"frontend_event_exception_state.handle_frontend_exception"` :x:
* `"state.frontend_event_exception_state.handle_frontend_exception"` ✅

Reply:
thanks! I just fixed it